software-modules/base/03-budgie/budgie.sh

```diff
@@ -136,19 +113,20 @@ rm /usr/lib/udev/rules.d/*udisks2*.rules
 
 ## Compile schemas to include connman-gtk
 glib-compile-schemas /usr/share/glib-2.0/schemas/
-
 systemctl daemon-reload
 systemctl enable lightdm.service
-systemctl enable hsync.timer
-
```

Comment:
I'm not sure why `enable` does not work and shows `Failed to enable unit: Refusing to operate on alias name or linked unit file: hsinc.timer` although is neither an alias nor a linked file, but `start` does work